index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Lynn's portofolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+</head>
+<body>
+    <header>
+        <h2>John Doe</h2>
+
+        <nav>
+          <ul class="menu">
+            <li><a href="#">Home</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contacts</a></li>
+          </ul>
+        </nav>
+      </header>
+      
+    <main class="main">
+        <h1>I'm Laura Jong </h1>
+         <p>Software Developer</p>
+        <p>I can help you  to build a products , feature or website Look through some of my work and experience!<br/> If you like what you see and have a project you need coded,  don’t hesitate to contact me.</p>
+
+   </main>
```

Comment:
This is a section, as the tag above is the header, this one should be a section, not the main